index.html

```diff
@@ -0,0 +1,128 @@
+          </div>
+          <nav class="nav">
+           <ul class="toggle">
+                <li><a href="./about.html" class="">About</a> </li>
+                <li><a href="#" class="">Program</a> </li>
+                <li><a href="#" class="">Courses</a> </li>
+                <li><a href="#" class="start-learning" id="start-learning">Start Learning</a></li>
+             </ul>
+          </nav>
+        </div>
+      </header>
+      <main>
+        <section class="intro">
+          <h6>"Hello ! Welcome"</h6>
+          <h1 class="tittle">  E-learning MODERN WAY OF <br> LEARNING 2022, <br>2023 </h1>
+          <div class="invite-wrap">
+            <p>Study your Courses anytime and anywhere</p>
+            <p>Access the study material unlimited times with a quick mode of delivery</p>
+          </div>
+          <h2 class="a-date">22nd Nov 2022 (THU) To <img src="images/infinity.png" id="infinity"></img></h2>
+          <p class="adrasha">@ International Conference on Climate Change (ICCC)</p>
+        </section>
+        <section class="main-program">
+          <h2>Main Courses</h2>
+          <div class="bar-icon"></div>
+          <div class="card-container">
+            <div class="card">
+              <img class="program-icon" src="images/program_icon_01.png" alt="">
+              <h3 class="program-tittle">Lecture</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit.</p>
+            </div>
+            <div class="card " >
+              <img class="program-icon" src="images/program_icon_02.png" alt="">
+              <h3 class="program-tittle program-tittleNoWrap">CC Exhibition</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit.</p>
+            </div>
+            <div class="card ">
+              <img class="program-icon" src="images/program_icon_03.png" alt="">
+              <h3 class="program-tittle">Forum</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit.</p>
+            </div>
+            <div class="card ">
+              <img class="program-icon" src="images/program_icon_04.png" alt="">
+              <h3 class="program-tittle">Workshop</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit.</p>
+            </div>
+            <div class="card ">
+              <img class="program-icon" src="images/program_icon_05.png" alt="">
+              <h3 class="program-tittle">CC Ignite</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum.</p>
+            </div>
+          </div>
+          <div>
+            <a href="#" id="see-program">SEE ALL COURSES</a>
+          </div>
+        </section>
+        <section class="introduction show" id="introduction">
```

Comment:
- [ ] [OPTIONAL] In the desktop view all the speakers should be shown automatically without the need to click on the `More` button, kindly consider making sure this is applied in your design

|Your page|Provided design|
|-|-|
|![Screenshot_20221203_102432](https://user-images.githubusercontent.com/25227022/205433835-60ca508f-0a79-42cd-8a74-80aa04d80508.png)|![Screenshot_20221203_093130](https://user-images.githubusercontent.com/25227022/205433841-753305f8-56b7-44de-9838-32d39345feb6.png)|